lib/src/implementations/prismic_io_article_repository.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import 'package:rxdart/rxdart.dart';
+import '../blocs/article_gettable.dart';
+import './prismic_io_article.dart';
+
+class PrismicIoArticleRepository implements ArticleGettable {
+  @override
+  ValueObservable<List<Article>> getArticles() {
+    final observable = BehaviorSubject<List<Article>>();
+
+    const url =
+        "https://saraka.cdn.prismic.io/api/v2/documents/search?ref=XMfh-RAAABUjzjv8";
```

Comment:
Is this for debugging? If it is so, delete it. Otherwise, give the value from outside of the class